From 826555276e994ea822646e280fc823118a53abfc Mon Sep 17 00:00:00 2001
From: Captain Polliwog build <build@captainpolliwog.invalid>
Date: Fri, 18 Sep 2026 17:08:29 -0400
Subject: [PATCH 35/46] Turn on User Timing and Resource Timing

performance.mark() and friends, which Safari 11 shipped enabled. Sites
call them unguarded (GitHub's scripts each mark their own parse time).
---
 Source/WebKitLegacy/mac/WebView/WebPreferences.mm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Source/WebKitLegacy/mac/WebView/WebPreferences.mm b/Source/WebKitLegacy/mac/WebView/WebPreferences.mm
index c15ecef5..ddc7d32d 100644
--- a/Source/WebKitLegacy/mac/WebView/WebPreferences.mm
+++ b/Source/WebKitLegacy/mac/WebView/WebPreferences.mm
@@ -681,8 +681,8 @@ public:
         [NSNumber numberWithBool:NO], WebKitIntersectionObserverEnabledPreferenceKey,
 #endif
         [NSNumber numberWithBool:NO], WebKitDisplayContentsEnabledPreferenceKey,
-        [NSNumber numberWithBool:NO], WebKitUserTimingEnabledPreferenceKey,
-        [NSNumber numberWithBool:NO], WebKitResourceTimingEnabledPreferenceKey,
+        [NSNumber numberWithBool:YES], WebKitUserTimingEnabledPreferenceKey,
+        [NSNumber numberWithBool:YES], WebKitResourceTimingEnabledPreferenceKey,
         [NSNumber numberWithBool:NO], WebKitCredentialManagementEnabledPreferenceKey,
         [NSNumber numberWithBool:NO], WebKitMediaUserGestureInheritsFromDocument,
         [NSNumber numberWithBool:NO], WebKitIsSecureContextAttributeEnabledPreferenceKey,
-- 
2.50.1 (Apple Git-155)

